src/meta/src/model/stream.rs

```diff
@@ -55,7 +55,7 @@ impl From<PbTableParallelism> for TableParallelism {
             Some(Fixed(FixedParallelism { parallelism: n })) => Self::Fixed(*n as usize),
             Some(Auto(_)) => Self::Auto,
             Some(Custom(_)) => Self::Custom,
-            _ => Self::Auto,
+            _ => Self::Custom,
```

Comment:
IIUC old tables should have `TableParallelism parallelism = 7;` unset, instead of `oneof parallelism` unset?

Reply:
Yes indeed, it should also be `unreachable`.